### PR TITLE
Join us careers page

### PR DESCRIPTION
**Pull request by a-wong-8**
https://deploy-preview-36--creative-jalebi-f8e18b.netlify.app/job-listing

### CODE REVIEW

**Review by a-wong-8 — COMMENTED** [review]
Screenshot 2024-06-25 at 19-28-44
Screenshot 2024-06-25 at 19-29-22

**Comment by gevuong** [comment]
Can you:
1. link this page to the Join Our Team button at the bottom of oyw page
2. change the url route to `/job-listing` so that it adheres to standard naming convention

**Comment by gevuong** [comment]
I just noticed a typo in screenshot, "Software engineer" should be "Software Engineer"

**Comment by ridham-virufy** [comment]
The topmost text overlaps with theVirufy logo in the navbar ar 320px width. 
image

Other than that everything seems perfect

**Comment by gevuong** [comment]
If a user selects no in the modal, I think it would make more sense to redirect them back to the `one-young-world` page since that was the entry point to `/job-listing`. Lmk what you think.

**Comment by a-wong-8** [comment]
ready for review

File: src/components/atoms/button/ButtonWhite.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+import { ButtonType, ButtonSize } from "../../themes";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export default function Button({ size, type, label, path, onClick = () => { } }) {
```
Comment (gevuong) [inline src/components/atoms/button/ButtonWhite.js]: `ButtonType` and `ButtonSize` imports, and `size` and `type` props do not appear to be used here..

Why is `onClick` needed here if it's not used as a prop in `job-list.js`?

**Comment by gevuong** [comment]
Do you get lint warnings if imports are unused? I'm also noticing that `useState` and `useEffect` is also unused in `job-list.js`. I suggest reviewing all the files you worked on and removing any unused imports, props, variables, etc.

File: src/components/atoms/button/ButtonWhite.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
```
Comment (gevuong) [inline src/components/atoms/button/ButtonWhite.js]: why is this needed?..

File: src/components/atoms/button/ButtonWhite.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+export default function Button({ label, path }) {
+
+    return (
+        <>
+            <button className={`bg-white rounded-3xl w-32 font-medium mr-4 h-8 my-auto`}><Link href={path}>{label}</Link></button>
+        </>
+    )
+}
```
Comment (gevuong) [inline src/components/atoms/button/ButtonWhite.js]: Does your editor auto format code on save? If not I recommend figuring out how to enable it. It can immediately improve readability and consistency.

Upon formatting on save, I get this:
```
export default function Button({ label, path }) {
  return (
    <>
      <button
        className={`my-auto mr-4 h-8 w-32 rounded-3xl bg-white font-medium`}
      >
        <Link href={path}>{label}</Link>
      </button>
    </>
  )
}
```

Comment (a-wong-8) [reply src/components/atoms/button/ButtonWhite.js]: when I select format I get this error: 
Screenshot 2024-06-26 at 5 06 01 PM

Comment (gevuong) [reply src/components/atoms/button/ButtonWhite.js]: Please install this: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

_Edit: Once it's working, save the files you worked on and it should auto format, then , make a new commit with those changes. Moving forward, I highly recommend getting this to work to enforce consistent styling across codebase._

Comment (a-wong-8) [reply src/components/atoms/button/ButtonWhite.js]: ok I got it working, thanks

**Comment by gevuong** [comment]
Wow it looks a lot better with prettier enabled, thanks for taking the time to figure that out.

It looks like the header element for each job category is smaller than the text within it, and is also labeled as an `<h1>`. Can you increase the text size so it looks more like a heading?

As a best practice, it is recommended to [avoid using multiple `<h1>` elements on one page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page), and for semantic purposes, those job headings would be more fitting with an h2 or h3 element.

Screenshot 2024-06-27 at 9 40 40 AM

**Review by gevuong — APPROVED** [review]
